projects/solar/src/data_store.c

```diff
@@ -22,6 +22,15 @@ StatusCode data_store_set(DataPoint data_point, uint32_t value) {
   return STATUS_CODE_OK;
 }
 
+StatusCode data_store_set_signed(DataPoint data_point, int32_t value) {
```

Comment:
You should add tests for these in test_data_store.c that just make sure they work correctly - we want to make sure there are no weird conversion errors.

Reply:
How is that different from the tests already put in place that converts `0x800` to `-59` and `28` respective to the thermistor type?

Reply:
The tests in test_sense_temperature.c would indirectly test this function through converting 0x800 to -59 or 28 yes, but we shouldn't rely on the sense_temperature tests to test data_store - the data_store tests should test all the functionality in that module, and other modules' tests should then just assume that data_store works. Though it looks like you've removed the signed functions so I suppose it's a moot point.